doc/private_endpoint_connection_command_guideline.md

```diff
@@ -0,0 +1,88 @@
+## Command Guideline
+
+#### Private Endpoint Connection
+
+- The parent resource should expose a single command group called `private-endpoint-connection` with four commands: `approve`, `reject`, `delete`, `show`.
+- If `approve` and `reject` commands are long running operations, please also provide `... private-endpoint-connection wait` command and support `--no-wait` in `approve` and `reject` commands.
```

Comment:
Typically it should be the same for delete.

Reply:
Sure. I will add delete too.